source/module_hamilt_lcao/module_deepks/LCAO_deepks.h

```diff
@@ -219,6 +228,10 @@ class LCAO_Deepks
     void init_orbital_pdm_shell(const int nks);
     void del_orbital_pdm_shell(const int nks);
 
+    //for v_delta label calculation; xinyuan added on 2023-2-22
```

Comment:
Please add detail annotation for new functions and try to add UnitTests for them.